### ISSUE TITLE
Big revision and big jump in Greece and Norway cumulative tests

### ISSUE THREAD

**Issue by DataOps-epam** [finddx/FINDCov19TrackerData#197]
Hi,

Could you please advise the nature of big revision for Norway cumulative tests from 2021-02-19 till 2022-08-23 and big jump for Greece cumulative tests for 2022-08-10 till 2022-08-23 period? Such changes looks suspicious.

Thank you.

**Comment by findanna** [comment]
@benubah @seawaR could you please check on this as well? Were these updated through csv files reporting past values for the above countries?

**Comment by benubah** [comment]
@seawaR  did a bulky update for Norway yesterday with the data from the new source (https://www.fhi.no/en/id/infectious-diseases/coronavirus/daily-reports/daily-reports-COVID19/) for the period 2021-02-19 to 2022-08-22, so there might have been revisions at the source for the stated period. This means the numbers might have changed for these days at the source and which is what is reflected in our data-set. No action needed here.

We're correcting Greece

**Comment by DataOps-epam** [comment]
Hi,

As we see, you sent other big revision for Norway cumulative tests from 2021-02-23 till 2022-08-24. Such change looks suspicious. Could you please confirm that previous revision on 08/23/2022 was bad?

**Comment by benubah** [comment]
The revision on 08/23/2022 was correct on that day. The reason we did a revision at our end for 2021-02-19 to 2022-08-22  is because Norway is no longer publishing cumulative tests like other countries we track. They now provide a daily log of new tests.

We are collecting data for Norway from this source (https://www.fhi.no/en/id/infectious-diseases/coronavirus/daily-reports/daily-reports-COVID19/#table-pagination-38550455) and we just verified that the data at this source is revised daily - this means that _some_ of the numbers we got on 08/23/2022 were revised on 08/24/2022 and some were again revised today at the source. It seems they get more test data for past days and then add this to the counts of previous days, and so numbers in the past keep changing.

We're working to fully automate scraping for this new source.
